### DISCUSSION TITLE
xd-AntiSpy UI

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/xd-AntiSpy#16] [Polls]
I've just finished completely porting xd-AntiSpy from C++ to C# and now I'm discovering something amusing. Turns out, at the client's place, there was still an old skin in CCleaner version 1 style :) Phew, that thing is really 15 years old, and I managed to apply it to the C# version of xd-AntiSpy. What do you think? Should we continue to stay true to the XD-AntiSpy UI in the future? Do we want to freeze the classic WinForms UI without any frills and never modernize it again? The CCleaner UI with a left-aligned list view and right-aligned log might not be as compact, but it's definitely more functional.

![xd-AntiSpy_A4GdPtXeKs](https://github.com/builtbybel/xd-AntiSpy/assets/57478606/47b46a69-18ae-4506-95e1-c39d1d8f41a4)

**Comment by ed-br** [comment]
I wouldn't change the old XP-AntiSpy, but I'm sure we can give it a slight refresh
